django_rq/queues.py

```diff
@@ -49,6 +49,7 @@ class DjangoRQ(Queue):
     def __init__(self, *args, **kwargs):
         autocommit = kwargs.pop('autocommit', None)
         self._autocommit = get_commit_mode() if autocommit is None else autocommit
+        self.exception_handlers = kwargs.pop('exception_handlers', None)
```

Comment:
Ok. About the failing tests, idk how to fix it because, once the exception handler comes in, i couldn't find how to put them in failed again if they fail. 

That's the problem with the test. The failed job never go to failed after a exception_handlers takes in. :( 

I would like some help about it , if possible. 

Thanks! ^^